src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy

```diff
@@ -351,4 +376,132 @@ class DebugSpec extends BaseSpec {
         and: "Response should not contain ext.warnings"
         assert !response.ext?.warnings
     }
+
+    def "PBS should return debug information and emit metrics when account debug enabled and verbosity detailed"() {
+        given: "Default basic generic bid request"
+        def bidRequest = BidRequest.defaultBidRequest
+
+        and: "Account in the DB"
+        def accountConfig = new AccountConfig(
+                metrics: new AccountMetricsConfig(verbosityLevel: DETAILED),
+                auction: new AccountAuctionConfig(debugAllow: true))
+        def account = new Account(uuid: bidRequest.site.publisher.id, config: accountConfig)
+        accountDao.save(account)
+
+        and: "Flash metrics"
+        flushMetrics(defaultPbsService)
+
+        when: "PBS processes auction request"
+        def response = defaultPbsService.sendAuctionRequest(bidRequest)
+
+        then: "Response should contain ext.debug"
+        assert response.ext?.debug
+
+        and: "Metrics account should be incremented"
+        def metricsRequest = defaultPbsService.sendCollectedMetricsRequest()
+        assert metricsRequest[ACCOUNT_REQUESTED_WITH_DEBUG_MODE_METRICS.formatted(bidRequest.accountId)] == 1
+        assert metricsRequest[AUCTION_REQUESTED_WITH_DEBUG_MODE_METRICS]
```

Comment:
We need to ensure that metrics are not duplicating, so:
assert metricsRequest[AUCTION_REQUESTED_WITH_DEBUG_MODE_METRICS] == 1

Reply:
also please add check for request and account_request metrics